src/services/doppler-contact-policy-api-client.ts

```diff
@@ -19,13 +19,21 @@ export interface AccountSettings {
   emailsAmountByInterval: number | null;
   intervalInDays: number | null;
   excludedSubscribersLists: SubscriberList[] | null;
+  timeRestriction: TimeRestriction | null;
 }
 
 export interface SubscriberList {
   id: number;
   name: string;
 }
 
+export interface TimeRestriction {
+  timeSlotEnabled: boolean | null;
```

Comment:
why it is nullable?

Reply:
The only reason was to have consistency with the definition for the [`active` field](https://github.com/FromDoppler/doppler-webapp/blob/master/src/services/doppler-contact-policy-api-client.ts#L18) in the AccountSettings definition, which corresponds to the configuration for the shipments quantity section:

![image](https://github.com/FromDoppler/doppler-webapp/assets/1985175/2dd72894-c03b-4897-bdb6-0ed4d7d5768c)